integration/tests/managed/managed_nodegroup_test.go

```diff
@@ -444,23 +444,18 @@ var _ = Describe("(Integration) Create Managed Nodegroups", func() {
 					).
 					WithoutArg("--region", params.Region).
 					WithStdin(testutils.ClusterConfigReader(clusterConfig))
-
 				Expect(cmd).To(RunSuccessfully())
-			})
-		})
 
-		Context("and updating a nodegroup", func() {
-			It("should update a nodegroup", func() {
-				clusterConfig := makeClusterConfig()
+				By("and upddating the nodegroup's UpdateConfig")
 				clusterConfig.ManagedNodeGroups = []*api.ManagedNodeGroup{
 					{
-						NodeGroupBase: &api.NodeGroupBase{
-							Name: initialNodeGroup,
-						},
 						Spot: false,
+						UpdateConfig: &api.NodeGroupUpdateConfig{
+							MaxUnavailable: aws.Int(1),
+						},
 					},
 				}
-				cmd := params.EksctlUpdateCmd.
+				cmd = params.EksctlUpdateCmd.
 					WithArgs(
 						"nodegroup",
 						"--config-file", "-",
```

Comment:
do we want to assert it worked? e.g. fetching the new config from the EKS API to see its set to `1`? We do some direct EKS API calls in other integration tests (`unowned` for sure)

Reply:
ooo yeah good idea, let's do that! I'll add it!

Reply:
done in https://github.com/weaveworks/eksctl/pull/3855/commits/2d5f9b706719cf0e24ed805b1d9e6148c7c41422 !